tests/console/yast2_lan.pm

```diff
@@ -65,6 +65,14 @@ sub run {
     }
 
     assert_screen 'test-yast2_lan-1';
+    # Do not set hostname via DHCP - poo#66775
+    if (is_opensuse) {
```

Comment:
I don't see why openSUSE should be special here. Can you elaborate?

Reply:
It was just to avoid additional needling for SLE (which I cannot do). But if this is fine to include SLE as well, I can drop the condition.

Reply:
Better ask the test module maintainer

Reply:
Ok. Updated.